assets/model_monitoring/components/src/action_analyzer/action_analyzer_identify_problem_traffic/run.py

```diff
@@ -174,23 +174,19 @@ def parse_debugging_info(root_span):
                 if not parent_id:
                     print("No look up span found, skip action analyzer.")
                     return None
-                # todo, change this function using utils
-                lookup_span = tree.get_span_tree_node_by_span_id(parent_id)
-                lookup_input = json.loads(lookup_span.input)
-                index_content = lookup_input["mlindex_content"]
+                index_span = tree.get_span_tree_node_by_span_id(parent_id)
+                index_input = json.loads(json.loads(index_span.attributes)["inputs"])
```

Comment:
this is really weird - calling json.loads twice... but maybe that is correct, just seems strange

Reply:
The raw log is a dict of a dict and serialized. After the first parse, the dict.value is still a jsonstring